components/strava/strava.app.js

```diff
@@ -1,21 +1,29 @@
 // Strava API app file
 const axios = require("axios");
-
+const { axios: PlatformAxios } = require("@pipedream/platform");
 module.exports = {
```

Comment:
I would also recommend the migration from CommonJS to ESM modules and renaming all files from `.js` to `.mjs` extension
```suggestion
export default {
```

Reply:
done